core/models.py

```diff
@@ -1,9 +1,12 @@
 from django.db import models
 
+from tinymce import models as tinymce_models
+
 
 class Event(models.Model):
     name = models.CharField(max_length=255)
     venue = models.ForeignKey("Venue", on_delete=models.PROTECT)
+    description = tinymce_models.HTMLField(max_length=1000, null = True, blank = True)
```

Comment:
smallest of small things -- this formatting doesn't seem to abide by Black e.g., it would be `null=True, blank=True` with no spaces before/after the equal signs

I think that if we settle/standardize on black it would make things easier (no git conflicts purely due to linting). would that be ok with you?

Reply:
Yep totally! Thanks for catching that!